### ISSUE TITLE
http://mokshaproject.net/ redirects to https://reactdom.com/graphql 

### ISSUE THREAD

**Issue by abitrolly** [mokshaproject/moksha#72]
http://mokshaproject.net/ points to https://reactdom.com/graphql - is that really intentional?

**Comment by frenzymadness** [comment]
Now it redirects to `http://iyfbodn.com/?dn=mokshaproject.net&pid=9POT3387I&pbsubid=c6dd6375-e97d-4ef7-b672-35e549b7796f&noads=http%3A%2F%2Fiyfbodn.com%2F%3Fdn%3Dmokshaproject.net%26skipskenzo%3Dtrue` and that address is blocked by uBlock.
The problem is that this URL is the Homepage on PyPI.

**Comment by abitrolly** [comment]
Doesn't look like this project is active anymore.

@ralphbean right?

**Comment by frenzymadness** [comment]
Unfortunately, some important components depend on it in Fedora.

**Comment by abitrolly** [comment]
@frenzymadness if you mean https://packages.fedoraproject.org/ then it was rewritten some time ago.

https://lists.fedoraproject.org/archives/list/infrastructure@lists.fedoraproject.org/thread/WWQG4RE5PSR5I2GND5SVWGMZRJNVRRPS/

**Comment by frenzymadness** [comment]
moksha-hub depends on moksha-common and fedmsg and module-build-service depends on moksha-hub. That's what I see in the results of repoquery in Fedora rawhide.

**Comment by frenzymadness** [comment]
> moksha-hub depends on moksha-common and fedmsg and module-build-service depends on moksha-hub. That's what I see in the results of repoquery in Fedora rawhide.

I mean, if this is not true in reality, we should fix the packaging in Fedora and orphan and then retire the abandoned projects from Fedora Linux.

**Comment by ralphbean** [comment]
Yes - this project isn't active any more. No commits in 5 years.
